modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/sort/BaseSortTestCase.java

```diff
@@ -34,6 +39,38 @@
  */
 public abstract class BaseSortTestCase extends BaseIndexingTestCase {
 
+	@Before
+	public void setUp() throws Exception {
+		super.setUp();
+	}
+
+	@Test
+	public void testCustomScoreSorting() throws Exception {
+		List<String> stringValues = new ArrayList<>();
+
+		stringValues.add("test value");
+		stringValues.add("test");
+
+		String fieldName = "testField";
+
+		addDocuments(
+			value -> DocumentCreationHelpers.singleText(fieldName, value),
+			stringValues.stream());
+
+		Sort[] sorts = {new Sort(scoreParameter, Sort.SCORE_TYPE, true)};
+
+		assertKeywordSearchOrder(fieldName, sorts, stringValues);
+
+		sorts = new Sort[] {new Sort(scoreParameter, Sort.SCORE_TYPE, false)};
+
```

Comment:
Local variable should preferrably not be reassigned - recommend to inline value in usage point directly